fxvsetup.h (DO_SETUP_XY): use the same linux-i386 code for djgpp.

diff -urNp Mesa-3.4.2/src/FX/fxvsetup.h Mesa-3.4.2x/src/FX/fxvsetup.h
--- Mesa-3.4.2/src/FX/fxvsetup.h	2000-02-09 21:13:13.000000000 +0200
+++ Mesa-3.4.2x/src/FX/fxvsetup.h	2016-03-31 23:03:00.000000000 +0300
@@ -83,21 +83,21 @@
 #ifdef FX_V2
 #  define DO_SETUP_XY 				\
   v[XCOORD]=win[0];				\
   v[YCOORD]=win[1]; 
 #else
 #ifdef DRIVERTS
 #  define DO_SETUP_XY 				\
   v[XCOORD]=win[0]+fxMesa->x_offset;		\
   v[YCOORD]=win[1]+fxMesa->y_delta;		
 #else
-#  if (defined(__linux__) && defined(__i386__)) || defined(macintosh)
+#  if (defined(__linux__) && defined(__i386__)) || defined(macintosh) ||defined(__DJGPP__)
 #    define DO_SETUP_XY {			\
        GLfloat t1 = win[0] + snapper;	\
        GLfloat t2 = win[1] + snapper;	\
        v[XCOORD] = t1 - snapper;			\
        v[YCOORD] = t2 - snapper;			\
      }
 #  else
 #    define DO_SETUP_XY {				\
         /* trunc (x,y) to multiple of 1/16 */		\
         v[XCOORD]=((int)(win[0]*16.0f))*(1.0f/16.0f);	\
